src/main/java/nextstep/core/subway/favorite/application/FavoriteService.java

```diff
@@ -33,20 +32,20 @@ public FavoriteService(FavoriteRepository favoriteRepository, PathFinderService
     }
 
     @Transactional
-    public Favorite createFavorite(FavoriteRequest request, Member member) {
+    public Favorite createFavorite(FavoriteRequest request, Long memberId) {
         if (!pathFinderService.isValidPath(convertToPathFinderRequest(request))) {
             throw new IllegalArgumentException("출발역과 도착역을 잇는 경로가 없습니다.");
         }
-        return favoriteRepository.save(createFavoriteEntity(request, member));
+        return favoriteRepository.save(createFavoriteEntity(request, memberId));
     }
 
     public List<FavoriteResponse> findFavorites(Member member) {
-        return convertToFavoriteResponses(favoriteRepository.findByMember(member));
+        return FavoriteConverter.convertToFavoriteResponses(favoriteRepository.findByMember(member.getId()));
     }
 
     @Transactional
     public void deleteFavorite(Long favoriteId, Member member) {
-        if (!member.isMine(findFavoriteById(favoriteId))) {
+        if (!member.isSameId(findFavoriteById(favoriteId).getMemberId())) {
```

Comment:
즐겨찾기로부터 memberId 필드를 가져와서 이를 비교하는 데 사용하고 있네요. 
Favorite 에 비교하는 메서드를 추가한다면 [묻지말고 시켜라!](https://origogi.github.io/oop/%EB%AC%BB%EC%A7%80-%EB%A7%90%EA%B3%A0-%EC%8B%9C%EC%BC%9C%EB%9D%BC/) 를 적용해볼 수 있을 것 같아요.


```suggestion
        Favorite favorite = findFavoriteById(favoriteId);
        if (!favorite.isOwn(member)) {
```

Reply:
이런 피드백 너무 좋은 것 같습니다. 👍 

> 묻지말고 시켜라!

명심하겠습니다!! 😄